### ISSUE TITLE
Selecting a "leaf" item within a VRT Nu menu yields an error.

### ISSUE THREAD

**Issue by tebokial** [add-ons/plugin.video.vrt.nu#765]
### To Reproduce ###
1. Go to e.g. Most Recent
2. Click on Most Recent
3. VRT Nu Error 

### Additional context ###

- Operating system: Raspbian Stretch
- Kodi version: 18.1
- Addon version:  2.3.4
- Using a VPN: no
- Country you are using the addon from: Belgiekske

### Log (if available) ###
```paste below
[kodi.log](https://github.com/add-ons/plugin.video.vrt.nu/files/4689912/kodi.log)
2020-05-27 17:25:09.053 T:1596764912   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.AttributeError'>
                                            Error Contents: 'NoneType' object has no attribute 'value'
                                            Traceback (most recent call last):
                                              File "/home/herman/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon_entry.py", line 14, in <module>
                                                run(argv)
                                              File "/home/herman/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon.py", line 336, in run
                                                plugin.run(argv)
                                              File "/home/herman/.kodi/addons/script.module.routing/lib/routing.py", line 130, in run
                                                self._dispatch(self.path)
                                              File "/home/herman/.kodi/addons/script.module.routing/lib/routing.py", line 141, in _dispatch
                                                view_func(**kwargs)
                                              File "/home/herman/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon.py", line 197, in recent
                                                VRTPlayer().show_recent_menu(page=page)
                                              File "/home/herman/.kodi/addons/plugin.video.vrt.nu/resources/lib/vrtplayer.py", line 263, in show_recent_menu
                                                self._favorites.refresh(ttl=ttl('direct' if use_favorites else 'indirect'))
                                              File "/home/herman/.kodi/addons/plugin.video.vrt.nu/resources/lib/favorites.py", line 41, in refresh
                                                xvrttoken = TokenResolver().get_xvrttoken(token_variant='user')
                                              File "/home/herman/.kodi/addons/plugin.video.vrt.nu/resources/lib/tokenresolver.py", line 81, in get_xvrttoken
                                                return self._get_new_user_xvrttoken()
                                              File "/home/herman/.kodi/addons/plugin.video.vrt.nu/resources/lib/tokenresolver.py", line 234, in _get_new_user_xvrttoken
                                                xsrf = next((cookie for cookie in cookiejar if cookie.name == 'XSRF-TOKEN'), None).value
                                            AttributeError: 'NoneType' object has no attribute 'value'
                                            -->End of Python script error report<--



```

**Comment by dagwieers** [comment]
This is due to a change at VRT NU that we discovered during automated testing every 30 minutes.
We fixed it quickly in #761, and created a more extensive fix in #762.
A new release (v2.3.5) was made and is waiting for approval by Team Kodi.
See: https://github.com/xbmc/repo-plugins/pull/2937

You can already get the new release from: https://github.com/add-ons/plugin.video.vrt.nu/releases

**Comment by JeanPoll** [comment]
Thanks a lot for the effort !
